Split/Network/Streaming/SplitBackgroundSynchronizer.swift

```diff
@@ -0,0 +1,189 @@
+//
+//  BackgroundSynchronizer.swift
+//  Split
+//
+//  Created by Javier Avrudsky on 03/03/2021.
+//  Copyright © 2021 Split. All rights reserved.
+//
+
+import Foundation
+import BackgroundTasks
+
+@objc public class SplitBackgroundSynchronizer: NSObject {
```

Comment:
remove if only `SplitBgSynchronizer` is used

Reply:
Done!